app/controllers/backend/outgoing_payment_modes_controller.rb

```diff
@@ -31,6 +31,7 @@ class OutgoingPaymentModesController < Backend::BaseController
       t.column :name
       t.column :cash, url: true
       t.column :with_accounting
+      # t.column :sepa
```

Comment:
See previous review on same line. Fixed as of now (56eebae), probably by 3da1d44.